pyproject.toml

```diff
@@ -3,17 +3,17 @@ name = "chatlas"
 description = "A simple and consistent interface for chatting with LLMs"
 dynamic = ["version"]
 readme = "README.md"
-requires-python = ">=3.9"
+requires-python = ">=3.10"
 dependencies = [
   "pydantic>=2.0",
   "jinja2",
   "rich",
+  "mcp>=1.2.1",
```

Comment:
This adds quite a few dependencies for a feature that feels advanced for most users. Could we make it an optional dependency (and address the 3.9 support issues that others have mentioned)?